Scripts/gen_Mix.sh

```diff
@@ -98,6 +99,8 @@ if [ $nbb == "one" ]; then
   echo '#include "Production/JobConfig/mixing/OneBB.fcl"' >> mix.fcl
 elif [ $nbb == "two" ]; then
   echo '#include "Production/JobConfig/mixing/TwoBB.fcl"' >> mix.fcl
+elif [ $nbb == "low" ]; then
+  echo '#include "Production/JobConfig/mixing/LowIntensity.fcl"' >> mix.fcl
 else
   echo "Must specify 'one' or 'two' booster batches"
```

Comment:
Do you want to update the error message to include 'low' as an option?

Reply:
gen_Mix has major changes in another PR Id rather not adjust this here.